src/main/java/com/questhelper/quests/thedepthsofdespair/TheDepthsOfDespair.java

```diff
@@ -0,0 +1,280 @@
+import com.questhelper.requirements.QuestRequirement;
+import com.questhelper.requirements.Requirement;
+import com.questhelper.requirements.SkillRequirement;
+import com.questhelper.requirements.conditional.ConditionForStep;
+import com.questhelper.requirements.conditional.Conditions;
+import com.questhelper.requirements.conditional.InInstanceCondition;
+import com.questhelper.requirements.conditional.ItemRequirementCondition;
+import com.questhelper.requirements.conditional.ZoneCondition;
+import com.questhelper.steps.ConditionalStep;
+import com.questhelper.steps.DetailedQuestStep;
+import com.questhelper.steps.ItemStep;
+import com.questhelper.steps.NpcStep;
+import com.questhelper.steps.ObjectStep;
+import com.questhelper.steps.QuestStep;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.runelite.api.Favour;
+import net.runelite.api.ItemID;
+import net.runelite.api.NpcID;
+import net.runelite.api.ObjectID;
+import net.runelite.api.QuestState;
+import net.runelite.api.Skill;
+import net.runelite.api.coords.WorldPoint;
+
+@QuestDescriptor(
+	quest = QuestHelperQuest.THE_DEPTHS_OF_DESPAIR
+)
+public class TheDepthsOfDespair extends BasicQuestHelper
+{
+	// Recommended
+	ItemRequirement weapon, dramenStaff, foodIfLowLevel, superEnergyOrStamina, xericsTalisman, skillsNecklace,
+		varlamoreEnvoy, royalAccordOfTwill;
+
+	ConditionForStep inVineryHouse, inArceuusLibrary, hasTheVarlamoreEnvoy, inCaves, inFirstAreaOfCaves,
+		inSecondAreaOfCaves, inThirdAreaOfCaves, downstairsInCaves;
+
+	QuestStep talkToLordKandur, talkToChefOlivia, talkToGalana, findTheVarlamoreEnvoy, readTheVarlamoreEnvoy,
+		enterCrabclawCaves, goThroughCrevice, stepOverSteppingStones, climbPastRocks, enterTunnelEntrance,
+		talkToArturHosidius, killSandSnake, searchChest, talkToLordKandurAgain;
+
+	Zone houseWestOfVinery, arceuusLibraryF1, arceuusLibraryF2, arceuusLibraryF3, crabclawCaves, crabclawCavesPart1A,
+		crabclawCavesPart1B, crabclawCavesPart2, crabclawCavesPart3, crabclawCavesDownstairs;
+
+	@Override
+	public Map<Integer, QuestStep> loadSteps()
+	{
+		loadZones();
+		setupRequirements();
+		setupConditions();
+		setupSteps();
+
+		Map<Integer, QuestStep> steps = new HashMap<>();
+
+		ConditionalStep goTalkToLordKandur = new ConditionalStep(this, talkToLordKandur, inVineryHouse);
+		steps.put(0, goTalkToLordKandur);
+
+		ConditionalStep goTalkToChefOlivia = new ConditionalStep(this, talkToChefOlivia, inVineryHouse);
+		steps.put(1, goTalkToChefOlivia);
+
+		ConditionalStep goTalkToGalana = new ConditionalStep(this, talkToGalana, inArceuusLibrary);
+		steps.put(2, goTalkToGalana);
+
+		ConditionalStep findAndReadTheVarlamoreEnvoy = new ConditionalStep(this, findTheVarlamoreEnvoy);
+		findAndReadTheVarlamoreEnvoy.addStep(hasTheVarlamoreEnvoy, readTheVarlamoreEnvoy);
+		findAndReadTheVarlamoreEnvoy.addStep(inArceuusLibrary, talkToGalana);
```

Comment:
This step should be removed as it stops findTheValamoreEnvoy step from being used.